### PR TITLE
javascript2 week2 homework

### PR DESCRIPTION
**Pull request by shobanamg**

### CODE REVIEW

File: javascript/javascript2/week2/homework/doubling/script.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+//Say you would like to write a program that doubles the odd numbers in an array and throws away the even number.
+//Rewrite the above program using map and filter don't forget to use arrow functions.
+let numbers = [1, 2, 3, 4];
+
+let newNumbers = numbers
+  .filter((number) => number % 2 !== 0)
+  .map((number) => number * 2);
```
Comment (Divyajg) [inline javascript/javascript2/week2/homework/doubling/script.js:7]: Good one 👍

File: javascript/javascript2/week2/homework/hyfbay/main.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  products.forEach((product) => {
+    const productList = document.createElement("li");
+    productList.classList.add("product-list");
+
+    productList.innerHTML = `<li class="product-name">${product.name}</li>
+    <li class="product-price">price: ${product.price}</li>
+    <li class="product-rating">Rating: ${product.rating}</li>`;
+    productUl.appendChild(productList);
+  });
+}
```
Comment (Divyajg) [inline javascript/javascript2/week2/homework/hyfbay/main.js:15]: renderProducts() is a nice logic.

**Review by Divyajg — APPROVED** [review]
I haven't run your code, but the code looks very clean, and logic is perfect.
It seems you understood the DOM concept well. 
Great job. Keep coding. 👍

**Comment by shobanamg** [comment]
Thanks for the review @Divyajg. I will merge the code then.

**Pull request merged**